lib/binaries/ie_driver.ts

```diff
@@ -10,7 +10,8 @@ export class IEDriver extends Binary {
   static os = [OS.Windows_NT];
   static id = 'ie';
   static versionDefault = Config.binaryVersions().ie;
-  static isDefault = false;
+  static isDefault32 = false;
+  static isDefault64 = false;
   static shortName = ['ie', 'ie32'];
```

Comment:
Would you imagine that. `shortName` is not used at all. I must have had it there for something and abandoned it. Removing all `shortName`'s. :confused: